plugins/com.google.cloud.tools.eclipse.test.dependencies/pom.xml

```diff
@@ -12,7 +12,46 @@
   <version>0.1.0-SNAPSHOT</version>
   <packaging>eclipse-plugin</packaging>
 
+  <!--
+    Dependencies that are then exported to OSGi bundles.
+    We pull junit and hamcrest from Orbit.
+  -->
   <dependencies>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.objenesis</groupId>
+      <artifactId>objenesis</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy</id>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>lib</outputDirectory>
+              <ignoredDependencies>
+                <!-- ignore ourselves; otherwise copies our Bundle-Classpath -->
+                <ignoreDependency>com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.test.dependencies</ignoreDependency>
+              </ignoredDependencies>
+              <!-- includeScope defaults to test -->
+              <excludeScope>runtime</excludeScope>
+            </configuration>
```

Comment:
I'll add a note to the pom to explain.  Also realized that this benefits from an m2e connector for the `maven-dependency-plugin` to be installed, so I'll add a note to README.md